src/auth/exception/select-unauthorized-error.exception.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class SelectUnauthorizedError {
```

Comment:
What does the "Select" means here?

Reply:
Revised.
[refactor: revise the filename definition which is use for user login unauthorized error](https://github.com/Cophr/backend/pull/10/commits/25dcd61f10387546b030b29c85a00ad8c94163ce)